tools/travis/sql_v0.sql

```diff
@@ -0,0 +1,537 @@
+# This file is revision 0 of the server SQL schema
```

Comment:
Is the idea that this schema is patched with all the "update" SQL queries we have to verify that the requests are well-formed?

Reply:
Yes, to make sure each update can be applied successively, and to make sure that the latest SQL update can be applied to the previous version